packages/server/graphql/mutations/addAzureDevOpsAuth.ts

```diff
@@ -0,0 +1,111 @@
+import {GraphQLNonNull, GraphQLID} from 'graphql'
+// import getRethink from '../../database/rethinkDriver'
+import {getUserId} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import AddAzureDevOpsAuthPayload from '../types/AddAzureDevOpsAuthPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {GQLContext, GQLMutation} from '../graphql'
+import {isTeamMember} from '../../utils/authorization'
+import standardError from '../../utils/standardError'
+import segmentIo from '../../utils/segmentIo'
+import upsertAzureDevOpsAuths from '../../postgres/queries/upsertAzureDevOpsAuths'
+import AzureDevOpsServerManager from '../../utils/AzureDevOpsServerManager'
+import getAzureDevOpsAuthsByUserId from '../../postgres/queries/getAzureDevOpsAuthsByUserId'
+
+const addAzureDevOpsAuth: GQLMutation = {
```

Comment:
-2 this should go into the `addTeamMemberIntegrationAuth` mutation.

Reply:
tagging @wdfox

Reply:
@wdfox  made this change